From 28665b4fa55f18760ade77602d69f892d86b0bbd Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 18 Jan 2016 16:59:42 +0100
Subject: [PATCH] dracut-functions.sh:fix check_vol_slaves() volume group name
 stripping

commit 466a59984a095f33993cffd5a3bea40826469b03 removed whitespace
stripping from lvm volume group names.

(cherry picked from commit 2fd8588da51a0e43684852dcdabe327aca684a9f)
---
 dracut-functions.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut-functions.sh b/dracut-functions.sh
index 9218779c..d71ff676 100755
--- a/dracut-functions.sh
+++ b/dracut-functions.sh
@@ -687,7 +687,7 @@ check_vol_slaves() {
         if [[ $_lv = $2 ]]; then
             _vg=$(lvm lvs --noheadings -o vg_name $i 2>/dev/null)
             # strip space
-            _vg=$(printf "%s\n" "$_vg")
+            _vg="${_vg//[[:space:]]/}"
             if [[ $_vg ]]; then
                 for _pv in $(lvm vgs --noheadings -o pv_name "$_vg" 2>/dev/null)
                 do
